src/tree/workspace.rs

```diff
@@ -274,6 +274,12 @@ impl Node for WorkspaceNode {
     fn node_do_focus(self: Rc<Self>, seat: &Rc<WlSeatGlobal>, direction: Direction) {
         if let Some(fs) = self.fullscreen.get() {
             fs.tl_into_node().node_do_focus(seat, direction);
+        } else if let Some(stacked) = self.stacked.first() {
```

Comment:
Iirc, this list is sorted from bottom to top. Maybe it would be better to focus the topmost window?

Reply:
Agreed